ovos_config/locations.py

```diff
@@ -16,7 +16,7 @@
 from time import sleep
 import ovos_config.meta as _ovos_config
 from ovos_utils.xdg_utils import xdg_config_dirs, xdg_config_home, xdg_data_dirs, xdg_data_home, xdg_cache_home
-
+from ovos_utils.log import LOG
```

Comment:
why was this import added? is it used somewhere below not flagged by the github diff?

Reply:
Nope, leftover from debugging